frontend/scripts/react-components/shared/newsletter/newsletter.actions.js

```diff
@@ -4,9 +4,12 @@ import axios from 'axios';
 export const NEWSLETTER__SET_SUBSCRIPTION_MESSAGE = 'NEWSLETTER__SET_SUBSCRIPTION_MESSAGE';
 export const NEWSLETTER__RESET_NEWSLETTER = 'NEWSLETTER__RESET_NEWSLETTER';
 
-export const sendSubscriptionEmail = email => dispatch => {
+export const sendSubscriptionEmail = ({ email, firstname, lastname, organisation }) => dispatch => {
```

Comment:
I think we could do better error management (The errors are not showing correctly). And also some kind of success message. I'm not sure if this is part of this task but I don't think we will do it at any other time if we don't do it now

Reply:
good point, talked with pablo and we will copy what they do on finance, stay tuned;

Reply:
improved to a certain point!